albumartdb-com/monoalbum
# Logging

To test:
* ./deploy
* go to localhost:5601
* click on the three bar button ("hamburger") in the top left and select "logs" in the menu, and then select "stream live"
* then in a separate window go to localhost:3000 and open up the dev window
* then search for "train"
* you should see the result show up in the dev window showing the request returned successfully
* then you should see flask's log output for a successful API request on the backend show up in the log output

yay for increased visibility!

this branch has been fast forwarded to master to avoid conflicts
